canonical/ubuntu.com
# [pro update] Copy update on /desktop/developers

## Done

- Copy update on /desktop/developers

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/developers or https://ubuntu-com-11823.demos.haus/desktop/developers
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#heading=h.q5r79xcdu7p7)

## Issue / Card

Fixes #https://github.com/canonical-web-and-design/web-squad/issues/5618
